shared/actions/teams/creators.js

```diff
@@ -0,0 +1,12 @@
+// @flow
+import * as Constants from '../../constants/teams'
+
+function getChannels(teamname: string): Constants.GetChannels {
+  return {payload: {teamname}, type: 'teams:getChannels'}
+}
+
+function toggleChannelMembership(teamname: string, channelname: string): Constants.ToggleChannelMembership {
```

Comment:
I think this would be better if it took just `ConversationIDKey`

Reply:
mentioned above